onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -1230,15 +1231,20 @@ def _aten_convolution_onnx(
     input: TFloat,
     weight: TFloat,
     bias: TFloat,
+    transposed: BOOL,
     strides: Sequence[int],
     pads: Sequence[int],
     dilations: Sequence[int],
-    transposed: bool = False,
     output_padding: Sequence[int] = (0,),
     groups: int = 1,
 ) -> TFloat:
     """ConvXd with attributes pre-computed to fit the ONNX spec."""
 
+    # NOTE: transposed must be an input because when provided as an attribute,
```

Comment:
Is it correct to understand this is also a limitation on onnxscript side? (or onnx because no bool type attribute?)

Reply:
It's because onnx has no bool type attribute. We _may_ be able to allow the translator to recognize this as a special case and add a "cast to BOOL" when it's promoting it into an input? @gramalingam